app/src/components/GrantCard.vue

```diff
@@ -71,8 +71,10 @@ export default defineComponent({
   },
   components: { CartIcon },
   setup(props) {
-    const grantId = ref<any>(props.id);
+    const grantId = ref<BigNumberish>(props.id);
```

Comment:
`grantId` type should be a number here (and everywhere else, though I'm not sure if we've been consistent). Typically `BigNumberish` type should only be for function inputs, and variables/returned values should have a stricter type, e.g. we wouldn't want `grantId` to be represented as `Bytes` here just to cast to a number in the HTML template

Reply:
I've made this change! 🙌